src/constants/protocol-module.type.ts

```diff
@@ -9,5 +11,5 @@ export type ModuleAction = {
   description: string
   actions: InfinitActionRecord
   libPath: keyof typeof dependencies
-  Verifier: typeof BaseContractVerifier
+  Verifier: typeof TokenContractVerifier | typeof AaveV3ContractVerifier | typeof UniswapV3ContractVerifier
```

Comment:
Why? Each one should have the same interface?

Reply:
fixed